core/src/main/java/io/questdb/log/LogFactory.java

```diff
@@ -38,7 +38,7 @@
 public class LogFactory implements Closeable {
 
     public static final LogFactory INSTANCE = new LogFactory();
-    public static final String DEBUG_TRIGGER = "ebug";
+    public static final String DEBUG_TRIGGER = "debug";
```

Comment:
this was "ebug" on purpose. This is to deal neatly with `-Debug` command line option. We this change command line becomes `-Ddebug`

Reply:
I use IDEA's `Runner > Properties` feature of Maven run configurations instead of command line.